src/main/java/uk/ac/ebi/eva/pipeline/EvaPipelineJobLauncher.java

```diff
@@ -0,0 +1,185 @@
+import uk.ac.ebi.eva.pipeline.runner.exceptions.UnrecognizedElement;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Properties;
+
+@Component
+public class EvaPipelineJobLauncher extends JobLauncherCommandLineRunner {
+
+    private static final Logger logger = LoggerFactory.getLogger(EvaPipelineJobLauncher.class);
+
+    public static final String SPRING_BATCH_JOB_NAME_PROPERTY = "spring.batch.job.names";
+    public static final String PROPERTY_FILE_PROPERTY = "properties.file";
+    private static final String PROPERTY_FORCE_NAME = "force.restart";
+
+    @Value("${spring.batch.job.names:#{null}}")
+    private String jobNames;
+
+    @Value("${" + PROPERTY_FILE_PROPERTY + ":#{null}}")
+    private String propertyFilePath;
+
+    @Value("${" + PROPERTY_FORCE_NAME + ":false}")
+    private boolean forceRestart;
+
+    @Autowired
+    private JobRepository jobRepository;
+
+    private JobParametersConverter converter = new DefaultJobParametersConverter();
+
+    public EvaPipelineJobLauncher(JobLauncher jobLauncher, JobExplorer jobExplorer) {
+        super(jobLauncher, jobExplorer);
+    }
+
+    @Autowired(required = false)
+    @Override
+    public void setJobParametersConverter(JobParametersConverter converter) {
+        this.converter = converter;
+        super.setJobParametersConverter(converter);
+    }
+
+    @Override
+    public void run(String... args) throws JobExecutionException {
+        try {
+            checkAllParametersStartByDoubleDash(args);
+
+            String[] processedArgs = removeStartingHypens(args);
+            String[] filteredArgs = filterLauncherOnlyParameters(processedArgs);
+
+            checkIfJobNamesHasBeenDefined();
+            checkIfPropertiesHaveBeenProvided(filteredArgs);
+
+            // Command line properties have precedence over file defined ones.
+            Properties properties = new Properties();
+            if (propertyFilePath != null) {
+                properties.putAll(getPropertiesFile());
+            }
+            properties.putAll(StringUtils.splitArrayElementsIntoProperties(filteredArgs, "="));
+
+            setJobNames(jobNames);
+
+            if (forceRestart) {
+                logger.info("Force restart against job '" + jobNames + "' with parameters: " + properties);
+                ManageJobsUtils.markLastJobAsFailed(jobRepository, jobNames,
+                        converter.getJobParameters(properties));
+            }
+            logger.info("Running job '" + jobNames + "' with parameters: " + properties);
+            launchJobFromProperties(properties);
+        } catch (NoJobToExecute | NoParametersHaveBeenPassed | UnexpectedFileCodification | FileNotFoundException |
+                UnexpectedErrorReadingFile | NoPreviousJobExecution | UnrecognizedElement e) {
+            logger.error(e.getMessage());
```

Comment:
Yes I second @jmmut I would prefer a non-zero exit code. _JobExecutionException_ might not be the right one because the problem is in the properties file and not in the job.